CHANGELOG.md

```diff
@@ -16,3 +16,15 @@
  * Initial pretty-printing of ASN.1 Strucutres
  * Massive ASN.1 builder DSL streamlining
  * More convenient explicit tagging
+
+
+### NEXT
+* COSE Support
+* Full RSA and HMAC Support
+* new interface `Asn1OctetString` to unify both ASN.1 OCTET STREAM classes
```

Comment:
I would keep it consistent and start with an uppercase letter

Reply:
Fixed and expanded